blog/community-update-wasmcloud-incubation-process/index.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: community-update-wasmcloud-incubation-process
+title: 'Community Update: wasmCloud Incubation Process'
+image: '/img/wasmcloud_large_social.png'
+date: 2024-05-28T09:00:00+05:00
+author: 'Taylor Thomas'
+author_profile: 'https://twitter.com/_oftaylor'
+description: "An update on wasmCloud's Incubation Process"
+draft: false
+---
+
+Hey wasmCloud Community!
+
+We wanted to give you an update on our journey toward [incubation status](https://github.com/cncf/toc/issues/1338) with the Cloud Native Computing Foundation (CNCF). As some of you might have noticed, the process has been taking a little longer than we initially anticipated. But don't worry–there’s a good reason for the delay. The CNCF Technical Oversight Committee (TOC) has been hard at work streamlining the entire incubation process and we're one of the first projects to go through this updated process. Although it has been a delay for us, it is going to be a win for the larger CNCF community now that they've adopted this more streamlined process.
+
+<!--truncate-->
+
+The updated process, detailed [here](https://github.com/cncf/toc/blob/main/process/README.md#how-to-apply-to-move-levels), brings several upsides we wanted to provide as positive feedback. For one, it introduces a clearer, more structured template and process for projects to move through different levels of maturity within the CNCF. Previously, applicants had to fill out a separate document in addition to a PR, which included many repetitive sections and lacked clarity. It was also unclear exactly who was supposed to help move things along and in what timeframe. With the new issue template, many of these redundancies have been removed and simplified, minimizing confusion and making the overall due diligence process more transparent. This is great for maintaining high standards and building trust within the CNCF ecosystem. Additionally, each step of the process is explicitly layed out in the new document, making it easier for new applicants to understand the process and what it entails.
```

Comment:
...brings several upsides we wanted to provide as positive feedback.

This sentence does not parse - reorder/clarify meaning.